README.md

```diff
@@ -30,6 +30,8 @@ Check the [dYdX developer docs](https://docs.dydx.exchange/#/api?id=orderbook) f
 
 ```python
```

Comment:
Can you separate this into different markdown sections instead of having this be one massive python block?

This will make it a lot easier to read and find what you're looking for, and will also make it very easy to add to the docs page